frontend/src/pages/destructionlist/detail/DestructionListDetail.tsx

```diff
@@ -381,34 +439,65 @@ export function DestructionListDetailPage() {
         </Modal>
       )}
 
-      {destructionList.status === "ready_to_delete" && (
-        <Modal
-          title="Zaken definitief vernietigen"
-          open={destroyModalOpenState}
-          size="m"
-          onClose={() => setDestroyModalOpenState(false)}
-        >
-          <Body>
-            <P>
-              U staat op het punt om {destructionListItems.count} zaken
-              definitief te vernietigen.
-            </P>
-          </Body>
-          <Body>
-            <Form
-              buttonProps={{
-                variant: "danger",
-              }}
-              fields={destroyModalFormFields}
-              labelSubmit={`${destructionListItems.count} zaken vernietigen`}
-              validate={validateDestroy}
-              validateOnChange={true}
-              role="form"
-              onSubmit={handleDestroy}
-            />
-          </Body>
-        </Modal>
-      )}
+      {destructionList.status === "ready_to_delete" &&
+        !destructionList.plannedDestructionDate && (
+          <Modal
+            title="Zaken definitief vernietigen"
+            open={destroyModalOpenState}
+            size="m"
+            onClose={() => setDestroyModalOpenState(false)}
+          >
+            <Body>
+              <P>
+                U staat op het punt om {destructionListItems.count} zaken in het
+                vernietigingsproces te plaatsen. Deze zullen over 7 dagen
+                definitief worden vernietigd.
+              </P>
+            </Body>
+            <Body>
+              <Form
+                buttonProps={{
+                  variant: "danger",
+                }}
+                fields={destroyModalFormFields}
+                labelSubmit={`${destructionListItems.count} zaken vernietigen`}
+                validate={validateDestroy}
+                validateOnChange={true}
+                role="form"
+                onSubmit={handleDestroy}
+              />
+            </Body>
+          </Modal>
+        )}
+      {destructionList.status === "ready_to_delete" &&
+        destructionList.plannedDestructionDate &&
+        destructionList.processingStatus === "new" && (
```

Comment:
This is the same check as in `isPlannedForDestruction` so maybe that function can be reused